auacm/app/modules/submission_manager/judge.py

```diff
@@ -77,10 +80,10 @@ def compile_submission(submission, uploaded_file):
     filename = uploaded_file.filename
     name, ext = filename.rsplit('.', 1)
```

Comment:
Should `ext` here also be `_` like in line 225? Doesn't change anything functionally, just and unused variable.